apis/python/tests/test_experiment_query.py

```diff
@@ -14,10 +14,10 @@
 from somacore import AxisQuery, options
 
 import tiledbsoma as soma
-from tiledbsoma import SOMATileDBContext, _factory, pytiledbsoma
+from tiledbsoma import ExperimentAxisQuery, SOMATileDBContext, _factory, pytiledbsoma
 from tiledbsoma._collection import CollectionBase
 from tiledbsoma._experiment import Experiment
-from tiledbsoma._query import Axis, ExperimentAxisQuery
+from tiledbsoma._query import Axis
```

Comment:
IMHO, all of these tests of "public" API should import in the same manner a user would import them (to catch this bug in the future).

In other words, do:
> from tiledbsoma import Axis

Reply:
@bkmartinjr I tried that and got an error. But I'm not seeing it now when I try it just now ... 🤔

Reply:
a failure would indicate some other issue, so I don't recommend the "workaround" ....

the top-level namespace of the package should just "work"

Reply:
Totally agreed!

Reply:
I think probably I did the import of `Axis` in the test file but didn't do the re-export in the `__init__.py` -- that would explain it

Anyway I have it in both files & all is well now

Thanks @bkmartinjr !